modules/ROOT/pages/design-ghs-about-github-sync.adoc

```diff
@@ -0,0 +1,24 @@
+= GitHub Synchronization
+
+API specification designers often use multiple authoring and version control tools. This can result in having to maintain several different specification versions. 
```

Comment:
```suggestion
API specification designers often use multiple authoring and version control tools. This can result in maintaining several different specification versions. 
```